Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -193,6 +194,9 @@ def Set(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
 def Test(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
     ShowMof('TEST', ConfigType, ConfigID, Contents, Ensure, ContentChecksum)
     retval = 0
+    if not os.path.exists(AGENT_SCRIPT_PATH):
```

Comment:
NPMAgentBinaryCap.sh(AGENT_SCRIPT_PATH) is not added as part of our dsc module and is present in Providers/Scripts/. As it would not be present until dsc itself is updated in a future release of omsagent, we have come up with an onboard script that will run as root and do the following
1. create NPMAgentBinaryCap.sh as omsagent user and place it in /opt/microsoft/omsconfig/Scripts
2. update sudoers file to provide it NOPASSWD for omsagent user

The Test method adds a check for the presence of this file to ensure that our onboard script has been run and proceeds only if it is present.